### PR TITLE
Add flake parts

### PR DESCRIPTION
**Pull request by roberth**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,6 +102,7 @@
 * [Cachix](https://cachix.org/) - Hosted binary cache service; free for open-source projects.
 * [devshell](https://github.com/numtide/devshell) - `mkShell` with extra bits and a toml config option to be able to onboard non-nix users.
 * [dream2nix](https://github.com/nix-community/dream2nix) - A framework for automatically converting packages from other build systems to Nix.
+* [flake.parts](https://github.com/hercules-ci/flake-parts) - Minimal Nix modules for Flakes: split your flakes into modules and get things done with community modules.
```
Comment (houstdav000) [inline README.md:105]: I like most of this description. Add the word "framework" after "Minimal Nix modules", and I think it'd be perfect.

File: README.md [hunk 2]
```diff
@@ -130,6 +131,7 @@
 ### Haskell
 
 * [cabal2nix](https://github.com/NixOS/cabal2nix) - Converts a Cabal file into a Nix build expression.
+* [haskell-flake](https://github.com/srid/haskell-flake) - A [`flake-parts`](https://flake.parts) Nix module for Haskell development.
```
Comment (houstdav000) [inline README.md:134]: Please remove the link from "flake-parts" to make it consistent with the rest. It's otherwise good.

**Review by houstdav000 — CHANGES_REQUESTED** [review]
Add a word or two here and remove that link in a description and I think it's good.

File: README.md (hunk 2 above)
Comment (roberth) [reply README.md]: Not a wiki. Got it.

**Review by houstdav000 — APPROVED** [review]
Looks good to merge now.

**Comment by roberth** [comment]
Thanks!

**Pull request merged**